### PR TITLE
Commit: def NewGame function

### PR DESCRIPTION
**Pull request by johnorbz**
Pull request

### CODE REVIEW

**Comment by aarongupit** [comment]
Comment Test

**Comment by the0thergeorge** [comment]
test comment. good job!

File: word-search.py [hunk 1]
```diff
@@ -92,4 +92,8 @@ def printWordList():
 
 def wordSearch():
     # game start!
+    return
```
Comment (the0thergeorge) [inline word-search.py:95]: test comment

File: word-search.py [hunk 2]
```diff
@@ -92,4 +92,8 @@ def printWordList():
...
+
+def NewGame():
+    #reset all settings
```
Comment (aarongupit) [inline word-search.py:98]: Comment Test

**Pull request closed without merging**